scripts/genappimage.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
```

Comment:
It looks like you can simply use `#!/bin/sh`, the only thing which needs to be changed is LOWERAPP. Note that with bash you need `#!/usr/bin/env bash`, other scripts are using that.